MediaBrowser.Model/Configuration/ServerConfiguration.cs

```
@@ -162,7 +163,36 @@ public class ServerConfiguration : BaseApplicationConfiguration
         public bool SkipDeserializationForBasicTypes { get; set; }
 
         public string ServerName { get; set; }
-        public string BaseUrl { get; set; }
+        public string BaseUrl
+        {
+            get => _baseUrl;
+            set
+            {
+                _baseUrl = value;
+				// Normalize the start of the string
+                if (string.IsNullOrWhiteSpace(_baseUrl))
+                {
+                    // If baseUrl is empty, set an empty prefix string
+                    _baseUrl = string.Empty;
+                }
+                else if (!_baseUrl.StartsWith("/"))
+                {
+                    // If baseUrl was not configured with a leading slash, append one for consistency
+                    _baseUrl = "/" + _baseUrl;
+                }
+                else
+                {
+                    // If baseUrl was configured with a leading slash, just return it as-is
+                    _baseUrl = _baseUrl;
```

shouldn't this strip all leading slashes then add one, to handle cases like `///foo/`?

---

I don't think we should be silently discarding such things. They should throw a "not found" to indicate that multiple slashes are bad. This would get into "trying to work around users doing dumb things" territory and I don't think that's justified.